app/src/main/java/org/simple/clinic/summary/compose/StatinNudgeView.kt

```diff
@@ -142,6 +148,54 @@ fun RiskProgressBar() {
   }
 }
 
+@Composable
+fun StainNudgeAddButtons(
+    modifier: Modifier,
+    statinInfo: StatinInfo,
+    addSmokingClick: () -> Unit,
+    addBMIClick: () -> Unit,
+) {
+  if (statinInfo.cvdRisk != null) {
```

Comment:
Technically we can pull this check into the parent composable, since there is no alternative layout in this composable function if it's false

Reply:
Done